php/libraries/NDB_Menu_Filter.class.inc

```diff
@@ -595,7 +595,7 @@ class NDB_Menu_Filter extends NDB_Menu
         $this->setTemplateVar('rowsPerPage', $rowsPerPage);
         $this->setTemplateVar('pageID', $pageID);
 
-        if (is_array($_GET['filter']['order'])) {
+        if (isset($_GET['filter']) && is_array($_GET['filter']['order'])) {
```

Comment:
`$_GET['filter']` needs to be checked to be an array, not just set.

Reply:
@driusan yah both are checked. But currently I can get an error since it can be undefined.

Reply:
Both aren't checked. `$_GET['filter']` is only checked to be set, `$_GET['filter']['order']` is checked to be an array. `$_GET['filter']` needs to be both set and an array for `$_GET['filter']['order']` to be valid regardless of whether or not `$_GET['filter']['order']` is an array.